app/services/katello/pulp3/repository.rb

```diff
@@ -136,6 +136,16 @@ def refresh_mirror_entities
         RepositoryMirror.new(self).refresh_entities
       end
 
+      def refresh_entities
+        href = repo.remote_href
```

Comment:
Is this whole thing also achieved by update_remote method above? Could we reuse that?

Reply:
Let me try something tho...